.gitignore

```diff
@@ -41,3 +41,9 @@ gpu_stats/target/
 # Outputs of tools/local_wandb_server.py
 tools/local_wandb_server.state.lock
 tools/local_wandb_server.state
+
+# run outputs
+wandb/latest-run
+wandb/run-*
+wandb/debug*
+Artifacts/*
```

Comment:
this change doesn't belong here.

Reply:
also what is the `Artifacts`?

Reply:
oh my bad, this accidentally got committed. thanks for catching that

Reply:
the `artifacts` directory is just my local dir where any downloaded artifacts get saved 😅